overlay/index.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Overlay Specification
+description: Overlay Specification
+layout: default
+---
+
+# Overlay Specification
+
+## Specification Versions
+
+{% include specification-version-list.md specification="overlay" %}
+
+## Schema Iterations
+
+_Note that while schemas can catch many errors, they are not guaranteed to catch all specification violations.  In the event of a disagreement between the schemas and the corresponding specification text, the specification text is presumed to be correct._
+
+{% include schema-iteration-list.md specification="overlay" %}
```

Comment:
Should this include the same language about schema dates and patch numbers as the others?  Even if there are not yet multiple patch releases or schemas it's good to not have to go back and add it later.

Reply:
Added the paragraph now.